android/app/src/main/kotlin/io/rebble/cobble/service/WatchService.kt

```diff
@@ -58,7 +58,7 @@ class WatchService : LifecycleService() {
 
         super.onCreate()
 
-        if (!bluetoothAdapter.isEnabled) {
+        if (bluetoothAdapter?.isEnabled ?: false == false) {
```

Comment:
No need for `?: false`˙here.

Reply:
Outdated since it was fixed in master branch already and rebased out of PR.